CONTRIBUTING.md

```diff
@@ -24,43 +106,70 @@ DeepChem uses these tools or styles for keeping our codes healthy.
 Before making a PR, please check your codes using them.
 You can confirm how to check your codes from [Coding Conventions](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html).
 
-## Document Conventions
+### Document Conventions
 
-DeepChem use [Sphinx](https://www.sphinx-doc.org/en/master/) to build [the document](https://deepchem.readthedocs.io/en/latest/index.html).
-The document is automatically built by [Numpy style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html#numpydoc-docstring-guide) in source codes and [Napoleon extension](http://www.sphinx-doc.org/en/stable/ext/napoleon.html).
-It means you must add or modify Numpy style docstrings if you modify our codes or add new codes.
-
-Before making a PR, please don't forget to modify not only codes but also docstrings.
+DeepChem use [Sphinx](https://www.sphinx-doc.org/en/master/) to build
```

Comment:
Typo: "DeepChem uses". This was in in the original too, but good for us to fix :)